tests/unit/s2n_ktls_test.c

```diff
@@ -36,16 +36,22 @@ static int s2n_test_setsockopt_noop(int fd, int level, int optname, const void *
     return S2N_SUCCESS;
 }
 
-static int s2n_test_setsockopt_eexist_error(int fd, int level, int optname, const void *optval, socklen_t optlen)
+static int s2n_test_setsockopt_tcp_error(int fd, int level, int optname, const void *optval, socklen_t optlen)
 {
-    errno = EEXIST;
-    return S2N_FAILURE;
+    if (level == S2N_SOL_TCP) {
+        errno = EINVAL;
```

Comment:
Why does this no longer return EEXIST?

Reply:
Before it returned EEXIST because we ignored EEXIST from the S2N_SOL_TCP setsockopt call, so it caused the S2N_SOL_TLS setsockopt call to fail instead. I switched to actually checking the level to determine whether the call should fail or not.